person.rb

```diff
@@ -9,10 +9,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] As I mentioned before, Please add Getters and Setters `attr_accessor` for `@rentals`, not Getters only `attr_reader` as you did.
Other than that you have done a great job.